packages/gateway/src/features/registration/root-resolvers.test.ts

```diff
@@ -1735,8 +1737,7 @@ describe('Registration root resolvers', () => {
       ).rejects.toThrowError('FHIR request failed: Some error from search')
     })
 
-    it("throws an error when the user doesn't have register scope", async () => {
-      fetch.mockResponseOnce(JSON.stringify({ unexpected: true }))
+    it('throws an error when the declaration is not assigned', async () => {
```

Comment:
should we also keep the 'throws an error when the user doesn't have register scope'? Or can the record even be assigned to you if you don't have a 'register' scope? Does it make sense to then even check for the `hasScope(authHeader, 'register')` in the 'markEventAsNotDuplicate' resolver ?

Reply:
Yeah I think we can. Adding one right away.